src/Subscriber.js

```diff
@@ -18,8 +18,31 @@ class Subscriber {
 
     this._retryMap = new Map()
 
-    this._callback = () => Promise.resolve()
     this._initializePromise = undefined
+
+    this.actions = new Map()
+  }
+
+  _callback (msg) {
+    let {action, data} = typeof msg === 'object' ? msg : {}
```

Comment:
Be aware that `typeof null === 'object'` is true too. And `null` can't be destructured.
Instead you can write:
```suggestion
    let {action, data} = msg || {}
```
Because we expect an object anyway, and `null` will fall back to `{}` in this condition.

Reply:
Oh yes, I forgot about that bug in JS. Fixed